lib/Activity/Provider.php

```diff
@@ -93,10 +93,7 @@ private function initActivityParser(IEvent $event, array $params): void {
 	 * @param IEvent $event
 	 */
 	private function setIcon(IEvent $event): void {
-		$path = $this->urlGenerator->imagePath(
-			Application::APP_ID,
-			'circles.' . $this->activityManager->getRequirePNG() ? 'png' : 'svg'
```

Comment:
Why is the check not needed anymore?

Reply:
we have no more png anymore since 22, and it seems to be fine having svg for the generated event